src/pydisagg/models.py

```diff
@@ -18,53 +18,25 @@ class RateMultiplicativeModel(DisaggModel):
 
```

Comment:
I noticed in transformations.py, you have the abstract class in the same module as the other classes. Why did you choose to have DisaggModel in a different module than your other model classes?

Reply:
Both of these choices were somewhat arbitrary, I'll split them both up for consistency?

Reply:
Fixed in the commit below.